src/zoneerder/components/ol-map.ts

```diff
@@ -242,14 +242,15 @@ export class OlMap {
       } else {
         const geometry = f.getProperties().name.includes('Cirkel') ? ol.geom.Polygon.fromCircle(f.getGeometry())
           : f.getGeometry();
-        coordinates.push(geometry.getCoordinates());
+        coordinates.push(geometry.getCoordinates()[0]);
```

Comment:
.push(...geometry.getCoordinates()),  is wss veiliger om ze allemaal over te nemen ipv enkel de eerste?

Reply:
Maar tzit in ne forEach eh, hij gaat over alles heen en die coordinates worden eerst leeggemaakt